logstash-core/lib/logstash/api/commands/default_metadata.rb

```diff
@@ -15,6 +15,10 @@ def all
            :ephemeral_id => service.agent.ephemeral_id,
            :status => "green",  # This is hard-coded to mirror x-pack behavior
            :snapshot => ::BUILD_INFO["build_snapshot"],
+           :pipeline => {
+             :workers => LogStash::SETTINGS.get("pipeline.workers"),
+             :batch_size => LogStash::SETTINGS.get("pipeline.batch.size"),
```

Comment:
while we're here, let's also add  `batch.delay`

Reply:
Added and tested. It appears to produce a valid value:

```
{
  "host": "Mikes-MacBook-Pro.local",
  "version": "8.0.0",
  "http_address": "127.0.0.1:9600",
  "id": "aa75f2e7-e944-47ff-be03-74e03074f640",
  "name": "Mikes-MacBook-Pro.local",
  "ephemeral_id": "0ebd35db-87eb-4f59-864f-92d018c178ec",
  "status": "green",
  "snapshot": null,
  "pipeline": {
    "workers": 12,
    "batch_size": 125,
    "batch_delay": 50
  }
}```